xclim/data/fr.json

```
@@ -948,23 +870,23 @@
   "FREEZETHAW_SPELL_FREQUENCY": {
     "title": "Fréquence des événements de gel-dégel quotidien",
     "abstract": "Nombre de périodes de gel-dégel. Ces périodes sont des séries de jours consécutifs où la température minimale quotidienne est sous un seuil et la température maximale quotidienne au-dessus d'un autre. En général, ces deux seuils sont 0°C.",
-    "description": "Fréquence {freq:f} des événements de gel-dégel quotidien : Tmax > {thresh_tasmax} and Tmin <= {thresh_tasmin} pour au moins {window} jour(s) consecutif(s).",
+    "description": "Fréquence {freq:f} des événements de gel-dégel quotidien : Tmax > {thresh_tasmax} et Tmin <= {thresh_tasmin} pour au moins {window} jour(s) consecutif(s).",
     "long_name": "Fréquence {freq:f} des événements de gel-dégel quotidien"
   },
   "FREEZETHAW_SPELL_MAX_LENGTH": {
-    "title": "Longueur maximale des événements de gel-dégel quotidien",
-    "abstract": "Longueur maximale des périodes de gel-dégel. Ces périodes sont des séries de jours consécutifs où la température minimale quotidienne est sous un seuil et la température maximale quotidienne au-dessus d'un autre. En général, ces deux seuils sont 0°C.",
-    "description": "Longueur maximale {freq:f} des événements de gel-dégel quotidien : Tmax > {thresh_tasmax} and Tmin <= {thresh_tasmin} pour au moins {window} jour(s) consecutif(s).",
-    "long_name": "Longueur maximale {freq:f} des événements de gel-dégel quotidien"
+    "title": "Durée maximale du nombre de jours consécutifs avec un gel-dégel",
```

```suggestion
    "title": "Durée maximale d'une période de jours consécutifs avec un gel-dégel",
```
''Durée maximale du nombre'' feels weird to me...